uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_leave-request-record-actions.scss

```diff
@@ -0,0 +1,30 @@
+leave-request-record-actions {
+  display: block;
+
+  .leave-request-record-actions__button {
+    display: block;
+    float: left;
+    margin-left: 0;
+
+    &:first-child {
+      margin-right: $crm-gap-medium;
+    }
+  }
+}
+
+@media screen and (max-width: $screen-xs-max) {
+  leave-request-record-actions {
+    .leave-request-record-actions__button {
+      width: 100%;
+
+      > .btn {
+        width: 100%;
+      }
+
+      &:first-child {
```

Comment:
see the "what if you have 3 buttons" question above

Reply:
Done.